skimage/transform/_thin_plate_splines.py

```diff
@@ -0,0 +1,302 @@
+import numpy as np
+import scipy as sp
+
+
+def tps_warp(
+    image,
+    src,
+    dst,
+    output_region,
+    interpolation_order=1,
+    grid_scaling=None
+):
+    """Return an array of warped images.
+
+    Define a thin-plate-spline warping transform that warps from the
+    src to the dst, and then warp the given images by
+    that transform.
+
+    Parameters
+    ----------
+    src: (N, 2) array_like
+        Source image coordinates.
+    dst: (N, 2) array_like
+        Destination image coordinates.
+    image: ndarray
+        Input image.
+    output_region: (1, 4) array
+        The (xmin, ymin, xmax, ymax) region of the output
```

Comment:
```suggestion
    output_region: tuple of integers
        The region ``(xmin, ymin, xmax, ymax)`` of the output
```

Next you could make this parameter optional by defaulting to the dimensions of the `image` parameter.